smdebug_rulesconfig/debugger_rules/_constants.py

```diff
@@ -1,3 +1,5 @@
 RULE_CONFIG_FILE = "rule_config_jsons/ruleConfigs.json"
-RULE_GROUPS_CONFIG_FILE = "rule_config_jsons/ruleGroups.json"
 COLLECTION_CONFIG_FILE = "rule_config_jsons/collections.json"
+
+SUPPORTED_DL_FRAMEWORKS = {"TENSORFLOW", "MXNET", "PYTORCH"}
```

Comment:
Do we support layer inputs/outputs for all frameworks? Rules such as DeadRelu or neuron saturation require activation outputs. I am not sure if they would work for older TF versions: if not should we provide an additonal error message?

Reply:
Maybe the best place to put an error message like that is in the rule itself?